lib/Dialect/TritonGPU/IR/Dialect.cpp

```diff
@@ -1979,14 +1983,20 @@ SmallVector<int64_t> NvidiaMmaEncodingAttr::getMMAv2Rep(ArrayRef<int64_t> shape,
                                                     warpsPerCTA[rank - 1]))};
   }
 }
+
 unsigned NvidiaMmaEncodingAttr::getTotalElemsPerThreadForOperands(
     ArrayRef<int64_t> shape, Type eltTy, int kWidth, int opIdx) const {
   auto shapePerCTA = getShapePerCTA(*this, shape);
   int warpsPerCTAM = getWarpsPerCTA()[0];
   int warpsPerCTAN = getWarpsPerCTA()[1];
   // H100
   if (isHopper()) {
-    return getTotalElemsPerThread(shape, eltTy);
+    assert(opIdx == 0);
+    auto instrMNK = getInstrShape();
+    auto wpt = getWarpsPerCTA();
```

Comment:
You can use `warpsPerCTAM` from just above.

Reply:
Addressed in new split PR - resolving. Doing the same for all other comments in this PR, as soon as they are resolved in the corresponding split PR.